library/std/src/sys/unix/cmath.rs

```diff
@@ -1,32 +1,31 @@
 #![cfg(not(test))]
 
-use libc::{c_double, c_float};
-
+// These symbols are all defined in `compiler-builtins`
```

Comment:
This is only the case on some platforms, right? On most platforms, these are directly provided by `libm`.

Reply:
I changed the comment to "These symbols are all defined by `libm`, or by `compiler-builtins` on unsupported platforms."